Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,73 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "with-keg-only", "Keg only install (Skip linking json-fortran.) Required to support multiple Fortran compilers."
+  option "without-make-test", "Skip running build-time tests (not recommended)"
+
+  depends_on "cmake" => :build
```

Comment:
:heavy_check_mark: Dependencies un-duped. `cmake` is really only a `:build` dependency, but the brewtestbot `brew test json-fortran` fails unless it is set to be a `:run` dependency. Any advice on better ways to treat this is welcome.